components/Navbar/MobileNavbar/MobileNavbar.tsx

```diff
@@ -47,7 +47,8 @@ const addToMetamask = async (provider: any): Promise<void> => {
 
 const MobileNavbar: React.FC<IProps> = () => {
   const [state, setState] = useState<State>('initial')
-  const [tokenBalance, setTokenBalance] = useState<number>(0)
+  const [ssdTokenBalance, setSsdTokenBalance] = useState<number>(0)
+  const [maticTokenBalance, setMaticTokenBalance] = useState<number>(0)
```

Comment:
what's useState(0) do? (lemme know if i should just googs this myself, but using your PR as selfish learning opportunities) hahahaa - is that just setting the default state?

Reply:
yeah exactly, sets initial state ✊